com.ca.lsp.cobol/lsp-service-cobol/src/main/java/com/ca/lsp/cobol/service/delegates/validations/AnalysisResult.java

```diff
@@ -42,7 +44,8 @@ public static AnalysisResult empty() {
         Collections.emptyMap(),
         Collections.emptyMap(),
         Collections.emptyMap(),
-        Collections.emptyMap());
+        Collections.emptyMap(),
+        null);
```

Comment:
not sure if might create strange results but I would recommend to use an empty outline object rather than a null reference